convert.go

```diff
@@ -0,0 +1,120 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+// Package nativeconverter provides containerd-native converter
```

Comment:
The package comment should be updated.

Reply:
I don't see a need for this to be added in the top level package. It is an image utility function which can operate entirely from a client instance. I realize there are a few other functions which we could categorize like that, but don't see that as a reason to continue to add more.

Reply:
Moved the function to `images/converter` pkg